### PR TITLE
Deny submitting community posts for non verified agencies, return pos…

### PR DESCRIPTION
**Pull request by Tokikko**
…ts only for verified agencies

### CODE REVIEW

**Review by stacysealky — CHANGES_REQUESTED** [review]
Thanks for catching this issue and fixing it, @Tokikko 
I just have 1 suggestion for making the error message more concise.

Also, shouldn't we only display the post message section to agency.isVerified, and hide the section for !agency.isVerified ? Open to your suggestions.

File: src/api/controller/community.ts [hunk 1]
```diff
@@ -29,6 +29,13 @@ export default class CommunityController extends BaseController {
 
 			const agency = await this.agencyRepository.getAgencyByUserId(user._id);
 
+			if (!agency?.isVerified) {
+				return this.handleError(
+					res,
+					"We'd love to help you share your post! To ensure a smooth experience for everyone, kindly make sure your agency is verified before submitting your post. Thank you for understanding!",
+				);
```
[the remark this answers is not in the document]
Comment (Tokikko) [reply src/api/controller/community.ts:36]: Thats a rude sounding message :D

[the remark this answers is not in the document]
Comment (stacysealky) [reply src/api/controller/community.ts:36]: But an error message is just an error message. The purpose of it is not to be nice, but to inform the user in the most concise manner, so they understand what error they made immediately. 
"Please make sure your agency is verified before submitting your post." doesn't sound rude to me 😄

**Comment by stacysealky** [comment]
> > 
> 
> Yeah that would be nice. But in any case its good to leave a check if the try to post something to the api directly (not likely but in case it happens)

Could you also hide the post message section for the !agency.isVerified?
Only if it's not a lot of work..

**Comment by Tokikko** [comment]
Yes i created a separated ticket for that

File: src/api/controller/community.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (Tokikko) [reply src/api/controller/community.ts:36]: Joke :D

**Review by stacysealky — APPROVED** [review]

**Pull request merged**